apps/server/src/modules/system/controller/dto/oauth-config.response.ts

```diff
@@ -1,9 +1,21 @@
 import { ApiProperty } from '@nestjs/swagger';
 
 export class OauthConfigResponse {
-	constructor(oauthConfigResponse: OauthConfigResponse) {
+	constructor(oauthConfigResponse: {
+		redirectUri: string;
+		tokenEndpoint: string;
+		responseType: string;
+		clientId: string;
+		provider: string;
+		jwksEndpoint: string;
+		authEndpoint: string;
+		scope: string;
+		logoutEndpoint: string;
+		grantType: string;
+		issuer: string;
+	}) {
```

Comment:
If you want write a less, you can use the TypeScript utility types like `Readonly` or other <https://www.typescriptlang.org/docs/handbook/utility-types.html>.

Reply:
Thanks for the hint :)